straxen/plugins/pulse_processing.py

```diff
@@ -197,11 +206,47 @@ def compute(self, raw_records, start, end):
                     pulse_counts=pulse_counts,
                     veto_regions=veto_regions)
 
+    
+@export
+@strax.takes_config(
+    strax.Option('n_he_pmts', track=False, default=752,
+                 help="Maximum channel of the he channels"),
+    strax.Option('record_length', default=110, track=False, type=int,
```

Comment:
I think this can be removed. The linked mode will require the same length for all detectors. Why would you expect the he channels to be different?

Reply:
First we had the Default.Recordlen

then I tried the ```
record_length = strax.record_length_from_dtype(
            self.deps['raw_records_he'].dtype_for('raw_records_he'))
dtype['records_he'] = strax.record_dtype(record_length )```